test/functional/wallet_bumpfee.py

```diff
@@ -134,6 +137,29 @@ def test_nonrbf_bumpfee_fails(peer_node, dest_address):
     assert_raises_rpc_error(-4, "not BIP 125 replaceable", peer_node.bumpfee, not_rbfid)
 
 
+def test_mempool_replacement_timeout(self):
+    self.stop_node(0)
+    err_msg = "Error: mempoolreplacementtimeout has to be a non negative number below or equal to mempoolexpiry (in seconds)"
+    self.nodes[0].assert_start_raises_init_error(["-mempoolreplacementtimeout=-1"], err_msg)
+    beyond_expiry = 60 * 60 + 1
+    self.nodes[0].assert_start_raises_init_error(["-mempoolexpiry=1",
+                                            "-mempoolreplacementtimeout=%d" % (beyond_expiry)], err_msg)
```

Comment:
Indentation seems a bit off here. Consider using `black` :-)

Reply:
Yes looks off, you are right.

What is black?

Reply:
`black` is the new hot Python equivalent of `go fmt`. It is currently taking the open source Python world by storm! :-)

Try: `pip3 install black && black file_to_format.py`

Reply:
@practicalswift black will reformat the whole file, I don't think this is something we desire right now or should ask contributors to do

Reply:
Also, leaving feedback about whitespace in a review without leaving any other non-style comments about the code is distracting to the author and other reviewers

Reply:
`black` was suggested as a way to easily obtain sane formatting for the block in question. Reformatting the whole file as part of this PR is obviously a very bad idea :-)

Reply:
fixed in 03fa5a1b421be6c07bc5fc33118961d6da93fb65